doc/config.md

```diff
@@ -150,6 +150,7 @@ First, layers must be added to `config/wv.json/layers` and `config/wv.json/layer
   - Create .md file in `config/metadata`. The *description* (i.e. layer description/metadata) identifier has to be added to the corresponding measurement JSON file.
 - Add Data Download for product, if it exists
   - Refer to [Data Download](https://github.com/nasa-gibs/worldview/blob/master/doc/data_download.md) and create JSON file in `config/wv.json/products`. The *product* identifier has to be added to the corresponding layer JSON file.
+  - Rebuild the configuration with `npm run build:config` for use by the application.
```

Comment:
The `npm run build:config` line would be good to have on its own at the end of this list rather than embedded in the Data Download bullet